common/fw/CMakeLists.txt

```diff
@@ -13,26 +13,19 @@ set(REALSENSE_FIRMWARE_URL "https://librealsense.intel.com" CACHE STRING
 
 string(REGEX MATCH "D4XX_RECOMMENDED_FIRMWARE_VERSION \"([0-9]+.[0-9]+.[0-9]+.[0-9]+)\"" _ ${ver})
 set(D4XX_FW_VERSION ${CMAKE_MATCH_1})
-message(STATUS "D4XX_FW_VERSION: ${D4XX_FW_VERSION}")
+#message(STATUS "D4XX_FW_VERSION: ${D4XX_FW_VERSION}")
 set(D4XX_FW_SHA1 8dc3b8a0b9fdf03410c15b30e04f57701985064a)
 set(D4XX_FW_URL "${REALSENSE_FIRMWARE_URL}/Releases/RS4xx/FW")
 
-
 string(REGEX MATCH "SR3XX_RECOMMENDED_FIRMWARE_VERSION \"([0-9]+.[0-9]+.[0-9]+.[0-9]+)\"" _ ${ver})
 set(SR3XX_FW_VERSION ${CMAKE_MATCH_1})
-message(STATUS "SR3XX_FW_VERSION: ${SR3XX_FW_VERSION}")
+#message(STATUS "SR3XX_FW_VERSION: ${SR3XX_FW_VERSION}")
```

Comment:
Removed all of the FW prints? Why?

Reply:
Not all.
They were redundant.
It looks nicer now:
```
Fetching recommended firmwares:
... https://librealsense.intel.com/Releases/RS4xx/FW/D4XX_FW_Image-5.14.0.0.bin
... https://librealsense.intel.com/Releases/SR300/FW/SR3XX_FW_Image-3.26.1.0.bin
... https://librealsense.intel.com/Releases/L5xx/FW/L51X_FW_Image-1.5.8.1.bin